Formula/trafficserver.rb

```diff
@@ -41,7 +43,8 @@ def install
       --localstatedir=#{var}
       --sysconfdir=#{etc}/trafficserver
       --with-openssl=#{Formula["openssl@1.1"].opt_prefix}
-      --with-tcl=#{MacOS.sdk_path}/System/Library/Frameworks/Tcl.framework
+      --with-tcl=#{Formula["tcl-tk"].opt_lib}
```

Comment:
FWIW, ATS has removed TCL dependency since 9.0.0. 
```
configure: WARNING: unrecognized options: --with-tcl
```

Reply:
Thanks; I removed the `tcl-tk` dependency.